client/src/js/components/bhServiceOrDepot/bhServiceOrDepot.html

```diff
@@ -41,6 +41,7 @@
   <div ng-if="$ctrl.requestorType.type_key === 'depot'">
     <bh-depot-select
       depot-uuid="$ctrl.uuid"
+      user="true"
```

Comment:
What does this do?  I can't see where this is used in the `bhDepotSelect` component.

Reply:
````
    if ($ctrl.user) {
      options.only_user = true;
    }
````
**In :/bhima-2.X/client/src/js/components/bhDepotSelect.js**
it allows to filter only the repositories defined for the user